### PR TITLE
Hotfix 1.7.7

### PR DESCRIPTION
**Pull request by MocanaAtBroad**
Hotfix contains latest changes to circleci build/build deploy setup.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,8 +1,434 @@
...
+            set -u
+            readvault.sh secret/pepper/${ENVIRONMENT}/v1/conf .data.gcp.serviceKey | gcloud auth activate-service-account --key-file=-
+            gcloud --quiet config set compute/zone ${GOOGLE_COMPUTE_ZONE}
+          working_directory: ~/repo
+      - run:
+          name: Update GAE dispatch file
+          command: |
+            set -u
+            gcloud app deploy --project "broad-ddp-${ENVIRONMENT}" "${ANGULAR_PROJECT_DIR_PATH}/output-config/dispatch.yaml"
+          working_directory: ~/repo/ddp-workspace
```
Comment (MocanaAtBroad) [inline .circleci/config.yml:145]: This chunk was picked up from previous hotfix @zyme can you confirm? And do we need to bring this to `develop`?

Comment (zyme) [reply .circleci/config.yml:145]: Yes, we need to bring this over as well.

File: config/dispatch.yaml.ctmpl [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+dispatch:
+{{if eq $environment "dev"}}
+  - url: "angio.dev.datadonationplatform.org/*"
+    service: angio
+  - url: "angio.dev.datadonationplatform.org/*"
+    service: mbc
+  - url: "angio.dev.datadonationplatform.org/*"
+    service: brain
+  - url: "angio.dev.datadonationplatform.org/*"
+    service: osteo
```
Comment (yufengwng) [inline config/dispatch.yaml.ctmpl:12]: These doesn't look right... shouldn't they be `dev.ascproject.org`, `dev.braincancerproject.org`, etc? But I guess this won't hinder us from pushing to prod...

File: config/pepperConfig.js.ctmpl [hunk 3]
```diff
@@ -93,7 +93,13 @@ var DDP_ENV = {
...
+        DDP_ENV['auth0ClientId'] = "U27zbNPjGPWhLfvLKLSalY1rTvOZblIH";
+        DDP_ENV['projectGAToken'] = "UA-108484823-1";
+   {{else if eq $environment "staging"}}
+       DDP_ENV['auth0ClientId'] = "CPpfKxQ7tUnG66WHhEXiGhVe8MrC5W1A";
+       DDP_ENV['projectGAToken'] = "UA-108484823-1";
   {{end}}
 {{end}}
 
-{{end}}{{end}}{{end}}{{end}}
+{{end}}
```
Comment (yufengwng) [inline config/pepperConfig.js.ctmpl:105]: Nice!

File: ddp-workspace/projects/sandbox-app/app.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+service: sandbox
```
Comment (yufengwng) [inline ddp-workspace/projects/sandbox-app/app.yaml:1]: Do we actually need to create a deployment for the sandbox?

**Review by yufengwng — COMMENTED** [review]
LGTM as far as I can tell

File: ddp-workspace/projects/sandbox-app/app.yaml (hunk 4 above)
Comment (zyme) [reply ddp-workspace/projects/sandbox-app/app.yaml:1]: Nah

File: config/dispatch.yaml.ctmpl (hunk 2 above)
Comment (zyme) [reply config/dispatch.yaml.ctmpl:12]: No, these urls have all changed after chatting with ropes.

Comment (zyme) [reply config/dispatch.yaml.ctmpl:12]: I mean "these urls are right".  Sorry!

**Pull request closed without merging**